### DISCUSSION TITLE
Error when trying to install any plugin

### DISCUSSION THREAD

**Discussion by alextibor** [Eugeny/tabby#8512] [Q&A]
Hello

When I try to install any plugin I get the following error:

```
Error in <plugin>:
[object Object]
``` 

![tabbyError](https://github.com/Eugeny/tabby/assets/26250155/3156e490-7048-4184-a82f-4a0f4ad75b7f)

Anyone have any idea what it could be?

**Comment by alextibor** [comment]
The error is im my system. Thanks!

**Reply by Cazus777** [reply]
how did you solve it?

**Comment by lvwenju** [comment]
I got the same problem, but can't solve it . any help , thanks!
